docs/api/embed.rst

```diff
@@ -0,0 +1,135 @@
+Embed API
+=========
+
+Read the Docs allows you to embed content from any of the projects we host.
+This allows reuse of content across sites, making sure the content is always up to date.
+
+.. contents::
+   :local:
+
+Usage
+-----
+
+Check our :doc:`/guides/embedding-content` guide to learn how to use the embed API in your projects.
+You can also use the API with our `sphinx-hoverxref`_ Sphinx extension.
+
+.. _sphinx-hoverxref: https://sphinx-hoverxref.readthedocs.io
```

Comment:
I'd like to remove this section because this document is about the raw API (similar to the regular APIv3 where we are not showing use cases). Let's keep consistency here.

If you want to cross-reference to our guide, you can use a note/tip with the first sentence linking to it. Besides, I wouldn't mention `sphinx-hoverxref` here since it's one of the topics the guide already talks about: we are duplicating content.